tests/new_api/conftest.py

```diff
@@ -47,3 +50,38 @@ def app(app):
     app.register_blueprint(draft_bp)
     app.register_blueprint(draft_action_bp)
     return app
+
+
+@pytest.fixture
+def create_user(app, db):
```

Comment:
Comment Invenio-Accounts come with a utility function [create_test_user](https://invenio-accounts.readthedocs.io/en/latest/api.html#invenio_accounts.testutils.create_test_user)

Reply:
How did I miss this :smile: ? Using it. Thanks!